### PR TITLE
Mobile first responsive markup

### PR DESCRIPTION
**Pull request by dreyhiflden**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,551 @@
...
+  background: #50c3af;
+  color: #f5f5f5;
+}
+
+.btn--grey {
+  background: #d2d5da;
+  color: #f5f5f5
+}
+
+.icon-star:before {
```
Comment (carrot-e) [inline css/style.css]: Why not `.icon--star`?

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,551 @@
...
+  left: 0;
+  width: 24px;
+  height: 24px;
+}
+
+.section-padding {
+  padding: 70px 0;
+}
+
+.bg-color--grey {
```
Comment (carrot-e) [inline css/style.css]: Is there a block `.bg-color`?

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,551 @@
...
+
+.card__content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0 20px;
+}
+
+.card__title {
+  font-size: calc(12px + (16 - 12) * ((100vw - 300px) / (960 - 300)));
```
Comment (carrot-e) [inline css/style.css]: Can you explain what happens here, please?

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,551 @@
...
+  .header__title {
+    margin: 0 0 4vw 0;
+  }
+
+  .header__subtitle {
+    font-weight: bold;
+  }
+
+  .header__subtitle::before,
+  .header__subtitle::after {
```
Comment (carrot-e) [inline css/style.css]: Previously you used single `:` for pseudo-elements. Please correct here or there for consistency.

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,551 @@
...
+    flex-direction: row;
+    justify-content: space-between;
+    height: 60px;
+  }
+
+  .footer__copyright,
+  .footer__social-links {
+    padding: 0;
+  }
+}
```
Comment (carrot-e) [inline css/style.css]: Files should end with a newline.

File: css/style.css (hunk 3 above)
Comment (dreyhiflden) [reply css/style.css]: That do scale font-size from a minimum of 12px (at a 300px viewport) to a maximum of 16px (at a 960px viewport).

**Pull request merged**